docs/cookbook/modifying-a-template-in-administration.md

```diff
@@ -0,0 +1,133 @@
+# Modifying a Template in Administration
+
+In this tutorial, we'll show you how to proceed if you need to modify a twig template in the administration.
+We will demonstrate this procedure with the example How to display a product transfer status on the product detail page in the administration.
```

Comment:
I think that "How to display a product transfer status on the product detail page in the administration." could be header instead of "Adding the information into the template". If not, put it at least into some quotes.

Reply:
Where did you find "Adding the information into the template" exactly?

What exactly should be quoted?

And, we have a cookbook named "Adding New Attribute to an Entity". This cookbook is not named "How to display an extId of a product on the product detail page in the administration". Do you see the difference? This cookbook is not created just for the scenario "product transfer status on the product detail page". This scenario is only the practical example.

Reply:
"Adding the information into the template" is next heading after this paragraph on line 6...

And if you do not want to do it at least quoute "How to display a product transfer status on the product detail page in the administration"

Reply:
quoted